src/registry_schemas/example_data/schema_data.py

```diff
@@ -2701,6 +2701,19 @@
     'freeze': True
 }
 
+AGM_LOCATION_CHANGE = {
+    'year': '2023',
+    'newAgmLocation': {
+        'streetAddress': 'mailing_address - address line one',
+        'streetAddressAdditional': '',
+        'addressCity': 'mailing_address city',
+        'addressCountry': 'CA',
+        'postalCode': 'H0H0H0',
+        'addressRegion': 'ON',
+        'additionalLocationDetails': ''
```

Comment:
can we update the sample data to use the delivery instructions field?

Reply:
I updated